### PR TITLE
crypto-xml: test behavior change caused by removing CAS related ctor

### PR DESCRIPTION
**Pull request by krwq**
Fixes: https://github.com/dotnet/corefx/issues/19272
Explanation: https://github.com/dotnet/corefx/issues/19272#issuecomment-299979378

### CODE REVIEW

**Comment by krwq** [comment]
@bartonjs PTAL

File: src/System.Security.Cryptography.Xml/tests/EncryptedXmlTest.cs [hunk 1]
```diff
@@ -633,7 +633,12 @@ public void DecryptData_CipherReference_InvalidUri()
                 ed.CipherData = new CipherData();
                 ed.CipherData.CipherReference = new CipherReference("invaliduri");
 
-                Assert.Throws<CryptographicException>(() => exml.DecryptData(ed, aes));
+                // https://github.com/dotnet/corefx/issues/19272
+                Action decrypt = () => exml.DecryptData(ed, aes);
+                if (PlatformDetection.IsFullFramework)
+                    Assert.Throws<ArgumentNullException>(decrypt);
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Xml/tests/EncryptedXmlTest.cs]: I'm sure this is an oops, but at least it doesn't work.

**Comment by safern** [comment]
Can we merge? Trying to get a clean desktop run ASAP.

**Comment by bartonjs** [comment]
@safern Good with me, sure.

**Comment by safern** [comment]
Thanks :)

**Pull request merged**